src/primitives/transaction.h

```diff
@@ -28,6 +28,9 @@ static const int SERIALIZE_TRANSACTION_NO_WITNESS = 0x40000000;
 // Globals to avoid circular dependencies.
 extern bool g_con_elementsmode;
 
+// SEQUENTIA:
```

Comment:
Remove comment Sequentia. We'll put this in elements part as a feature.

Reply:
Done in: https://github.com/SequentiaSEQ/SEQ-Core-Elements/pull/6/commits/0d0ec930a8d190886171a5478b7bc5a589258d96